src/languages/en-US.ts

```diff
@@ -72,14 +47,11 @@ export default {
 
 	global_logging_command: "`[{{timestamp}}]` {{emote_wrench}} **{{user.tag}}** `({{user.id}})` has executed `{{command}}` in <#{{channel}}>.",
 	global_no_reason: "No reason given",
-	// TODO: Remove, not needed anymore because of interaction deferring
-	global_loading: "{{emote_loading}} Loading...",
 	global_not_in_voice: "{{emote_fail}} User **{{target.tag}}** `({{target.id}})` is currently not connected to any Voice Channels!",
 	global_error: {
 		unknown: "{{emote_fail}} An unknown error has occurred, please try again later or if the issue persists contact the developers at {{discord_invite}}",
 		automod_items_length_undefined: "{{emote_fail}} An error occurred while attempting to parse the selected Automod items. No Automod items have been provided or Automod items length is undefined.",
-		// TODO: Rename to global
-		inviteinfo_guild_or_member_null:
+		global_guild_or_member_null:
```

Comment:
The name says "global" but the text says "parse the Invite"?

Reply:
Renamed this string as it originally popped-up in inviteinfo but I later on realized the same error happens in multiple places, hence the global rename

Reply:
It's pops up in multiple places, about parsing invites?